operations/kube-manifests/util/generator.jsonnet

```diff
@@ -0,0 +1,67 @@
+// The jsonnet file used to generate these kube-manifests
```

Comment:
can we get a readme with the command to generate the yamls from this jsonnet?

Reply:
added!